frontend/packages/erd-core/vitest.config.ts

```diff
@@ -1,11 +1,15 @@
 // biome-ignore lint/correctness/noNodejsModules: Because this file is a config file
 import * as path from 'node:path'
+import react from '@vitejs/plugin-react'
 import { defineConfig } from 'vitest/config'
 
 export default defineConfig({
+  plugins: react(),
   test: {
     alias: {
       '@': path.resolve(__dirname, './src'),
     },
+    environment: 'happy-dom',
```

Comment:
I don't have a strong preference between jsdom and happy-dom, but I chose happy-dom because it's faster.

> happy-dom emulates browser environment by providing Browser API, and considered to be faster than jsdom, but lacks some API, uses [happy-dom](https://github.com/capricorn86/happy-dom) package
https://vitest.dev/guide/environment.html#test-environment

Reply:
I think so too 👍🏻